src/pages/Map.css

```diff
@@ -0,0 +1,70 @@
+.PinCard {
+  width: 250px;
+  height: 200px;
```

Comment:
Why are you setting a fixed  height? what if the notes are longer?
![image](https://user-images.githubusercontent.com/6980777/131327200-7db56604-dfdc-466d-a8ee-a4611aec84c7.png)

Look how the overflow outside the card. Either you set the overflow of the card to be `scroll` or you remove the fix height and let the height adapt to the content